File: spring/SpringProject1/src/main/java/org/tvscredit/main/BookMain.java
package org.tvscredit.main;

import org.springframework.beans.factory.BeanFactory;
import org.springframework.context.support.ClassPathXmlApplicationContext;
import org.tvscredit.model.Book;

public class BookMain {

	public static void main(String[] args) {

		BeanFactory factory = new ClassPathXmlApplicationContext("beans.xml");
		Book b =  factory.getBean("book1",Book.class);
		
		System.out.println("Book Information ");
		
		System.out.println("isbn "+b.getIsbn());
		System.out.println("name "+b.getBname());
		System.out.println("cost "+b.getCost());
		System.out.println("author "+b.getAuthor());

	}

}

File: react/reactapp1/src/bookmain.js
import Author from "./Book/author";
import Publisher from "./Book/publisher";
import Reader from "./Book/reader";

export default function BookMain(){
    return(
        <div>
            <div style={{'margin-left':'400px'}}>
                <h1 style={{'margin-left':'0px'}} >Book Information</h1>
                  <div>ISBN {book.isbn} </div> 
                  <div>Name {book.bname} </div> 
            </div>
            <Author></Author>
            <Publisher></Publisher>
            <Reader></Reader>
        </div>
    )
}
var book={
    isbn:655665,
    bname:'two states',
}